File: common/python_common/mysql_utf8.py
# -*- coding: utf-8 -*-
#
#	python_common/mysql_utf8.py
#
#				Apr/21/2010
#
# --------------------------------------------------------
def	mysql_utf8_proc	(cursor):
	sql_str="SET NAMES utf8"
	cursor.execute (sql_str)
#
# --------------------------------------------------------

File: common/ruby_common/mysql_utf8.rb
#! /usr/bin/ruby
#
#	mysql_utf8.rb
#
#					Apr/13/2010
#
require	"cgi"
require	"json"
#
# ---------------------------------------------------------------------
def	mysql_utf8_proc	(dbi)
	$sql_str = "SET NAMES utf8";
	dbi.execute($sql_str);
end
# ---------------------------------------------------------------------
